local/bin/py/build/pull_rbac.py

```diff
@@ -31,18 +41,42 @@ def pull_rbac():
             print(f'\x1b[33mWARNING\x1b[0m: RBAC api request failure reason: {e}')
             pass
 
-    if json_result:
-        data = json_result.get('data', [])
+    if permissions_json and roles_json:
+        permissions_data = permissions_json.get('data', [])
+        # gets default roles (admin, standard, and read-only)
+        # ignores UI template roles ('Datadog Billing Admin Role' and 'Datadog Security Admin Role')
+        default_roles_data = roles_json.get('data', [])[0:3] 
+        dr_data_len = len(default_roles_data)
 
-        for permission in data:
+        for permission in permissions_data:
             group_name = permission['attributes']['group_name']
             permission_name = permission['attributes']['name']
+            permission_description = permission['attributes']['description']
+            permission_role_name = ''
+
+            for x in range(dr_data_len):
```

Comment:
Are we able to just loop over the data instead of doing a range here? or if we really need the index can we use enumerate?